resources/js/components/RelationshipFormItem.vue

```diff
@@ -117,17 +117,18 @@
             },
 
             fill(formData, parentAttrib) {
+	            formData.append(`${parentAttrib}[${this.id}][modelId]`, this.modelId);
                 this.getValueFromChildren().forEach(
                     (value, key) => {
                         let keyParts = key.split('_');
 
                         if (keyParts.length === 1) {
-                            formData.append(`${parentAttrib}[${this.id}][${key}]`, value);
+                            formData.append(`${parentAttrib}[${this.id}][values][${key}]`, value);
                         } else {
                             let parentParts = parentAttrib.split('_');
                             let attrib = keyParts.slice(parentParts.length + 1).join('_');
 
-                            formData.append(`${parentAttrib}[${this.id}][${attrib}]`, value);
+                            formData.append(`${parentAttrib}[${this.id}][values][${attrib}]`, value);
                         }
```

Comment:
Remove the else if possible. You could return at line 126 and remove the else, and return at line 131 too.